src/date-picker/date-display.jsx

```diff
@@ -131,15 +125,20 @@ const DateDisplay = React.createClass({
 
   render() {
     let {
+      DateTimeFormat,
+      locale,
       selectedDate,
       style,
       ...other,
     } = this.props;
-    let dayOfWeek = DateTime.getDayOfWeek(this.props.selectedDate);
-    let month = DateTime.getShortMonth(this.props.selectedDate);
-    let day = this.props.selectedDate.getDate();
-    let year = this.props.selectedDate.getFullYear();
-    let styles = this.getStyles();
+    const year = this.props.selectedDate.getFullYear();
+    const styles = this.getStyles();
+
+    const dateTimeFormated = new DateTimeFormat(locale, {
```

Comment:
minor, but `dateTimeFormatted` ?
Also, is there a use case to pass custom `options`? If so, how would that be achieved?